### ISSUE TITLE
Crash whilst attempting to combine an exact_count_distinct aggregate column on 0.9.9

### ISSUE THREAD

**Issue by andrew-blake** [pipelinedb/pipelinedb#1929]
I've created a stream that keeps track of sessions visiting our site. There is a continuous view with a 5 minute sliding window - chunked at 1 minute intervals.

I'm trying to aggregate those 1 minute counts and distinct count totals together.

Combining the count distinct totals is ok, but trying to combine an `exact_count_distinct` aggregate column causes a crash.

**Setup stream**

```
create foreign table source_stream
(
	id serial,
	timestamp_tz timestamp with time zone,
	sid text
) server pipelinedb;
```

**Setup continuous view**
```
create view stats_cv_1m with ( action = materialize, sw = '5 minutes') as
    select
          minute(timestamp_tz) as timestamp_1m
        , count(*) as cnt_1m
        , count(distinct sid) as cnt_unique_sids_1m
        , exact_count_distinct (sid) as cnt_unique_sids_1m_exact
    from source_stream
    group by 1
;
```

**Insert some test data**
```
insert into source_stream (id, timestamp_tz, sid) values (1, now(), 'aaabbccdd11');
insert into source_stream (id, timestamp_tz, sid) values (2, now(), 'aaabbccdd22');
insert into source_stream (id, timestamp_tz, sid) values (3, now(), 'aaabbccdd33');
insert into source_stream (id, timestamp_tz, sid) values (4, now(), 'aaabbccdd33');
insert into source_stream (id, timestamp_tz, sid) values (5, now(), 'aaabbccdd33');
```

**Inspect continuous view output**
```
select
    timestamp_1m
    , cnt_1m
    , cnt_unique_sids_1m
    , cnt_unique_sids_1m_exact
from stats_cv_1m;

      timestamp_1m      | cnt_1m | cnt_unique_sids_1m | cnt_unique_sids_1m_exact
------------------------+--------+--------------------+--------------------------
 2018-06-07 15:23:00+00 |      5 |                  3 |                        3
(1 row)
```

**Aggregate the 1 min HLL count distinct values into 5min totals**
```
select
      sum(cnt_1m) as cnt_5min
    , combine(cnt_unique_sids_1m) as cnt_unique_sids_5m
from stats_cv_1m;

 cnt_5min | cnt_unique_sids_5m
----------+--------------------
        5 |                  3
(1 row)
```

**Aggregate the 1 min `exact_count_distinct` values into 5min totals**
```
select
      sum(cnt_1m) as cnt_5min
    , combine(cnt_unique_sids_1m) as cnt_unique_sids_5m
    , combine(cnt_unique_sids_1m_exact) as cnt_unique_sids_5m_exact
from stats_cv_1m;

server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
The connection to the server was lost. Attempting reset: Succeeded.
```

**Comment by andrew-blake** [comment]
Will this crash be fixed in 1.0.0?

**Comment by andrew-blake** [comment]
I've confirmed this doesn't crash in 1.0.0. Thanks !
